applications/google_rise_2016.md

```diff
@@ -0,0 +1,414 @@
+
+**What makes your organization unique, innovative, and/or sets you apart from
+other groups doing similar work?**
+
+Our clubs and content are built and run by students, for students.
+
+By having high schoolers lead clubs, we're able to bypass the red tape typically
+associated with bringing computer science to a school or a community. Where it
+can take years to start a new computer science club, starting a Hack Club can
+take as little as two weeks. Our leaders don't need to find their own space (the
+school is their space), recruit external volunteers (their friends are
+volunteers), or hire teachers (they use our pre-written workshops). This model
+makes the cost of starting a new club virtually $0 from our end and we think
+it'll let us scale where others haven't.
+
+Our content is entirely open source and designed for community contribution.
+While we're still in the early stages of driving community contributions, half
+of our curriculum has been entirely built by the community. Having our content
+open source and accessible to contribution has the advantage of being
+self-correcting and continuously improved by club leaders and members.
+
+Additionally, our curriculum isn't built to teach people how to program:
+it's built to help our club members become self-taught programmers. Instead of
+starting by teaching the fundamentals, we throw our members in at the deep end
+by having them build real projects and use Google when they get stuck. By the
+end of the first club meeting, everyone has built their first website and
+deployed it using git, GitHub, and GitHub Pages.
+
+**Has this organization applied for, or received, funding from Google
+previously? If yes, please describe.**
+
+No
+
+### Primary Contact
+
+Please fill out the information below for the primary contact for this grant.
+The RISE Awards team would reach out to this primary contact with questions, if
+needed.
+
+**Full name of primary point of contact for this grant (first and last name)**
+
+Zachary Latta
+
+**Title of primary contact**
+
+Executive Director
+
+**Primary contact e-mail address**
+
+zach@hackclub.com
+
+**Primary contact telephone number (including international country code)**
+
+[TODO]
+
+### About the Project
+
+Please provide us with details about the specific computer science education
+project that you would like the RISE Awards to fund.
+
+**Project name**
+
+Hack Club (the whole organization)
+
+**Project description (one sentence):**
+
+Hack Club helps high schoolers start coding clubs through providing prospective
+club leaders with open-source materials and a community of other leaders around
+the globe.
+
+**Detailed project description (include all activities):**
+
+We would like RISE to fund the continued expansion of Hack Club. Specifically,
+we're looking to bring someone on full-time to lead the development of our
+curriculum with the community.
+
+To expand on what Hack Club does:
+
+[TODO]
```

Comment:
- plan and write curriculum
- maintain community guidelines
- onboard new clubs
- periodic check-ins with existing clubs over calls
- hack camp / other events
- monitor slack [almost] 24/7